app/__init__.py

```diff
@@ -5,15 +5,11 @@
 from flask_sqlalchemy import SQLAlchemy
 from flask_limiter import Limiter
 from flask_limiter.util import get_remote_address
+from app.healthcheck import add_health_check
 
 db = SQLAlchemy()
 migrate = Migrate()
 
-API_VERSION = "1.0"
-
-# healthcheck uses API_VERSION so this has to come after it's declared
-from app.healthcheck import add_health_check # noqa
```

Comment:
Side note, I'm super glad we were able to move this and remove the `# noqa` comment 😄